Some minor points that might catch out novices

WillEastbury opened the pull request:
(Or people working from bash on windows).

azurecla commented:
Hi __@stefsch__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (stefsch). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

WillEastbury committed:
Updating to catch gotcha if copying on bash/windows

also to add the -I parameter to specify the key

WillEastbury committed:
Make the port ranges explicit

The portal has both source and target port options for NSG filters, we should be specific about which one we set.

WillEastbury committed:
some minor corrections

GitHubber17 committed:
Update quick-create-portal.md
